web/packages/teleport/src/Discover/SelectResource/SelectResource.tsx

```diff
@@ -608,22 +623,43 @@ const Grid = styled.div`
   row-gap: 15px;
 `;
 
-const ResourceCard = styled.div<{ hasAccess?: boolean }>`
+const NewTabIconWrapper = styled.div`
   display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+  justify-content: flex-end;
+  position: absolute;
+  top: ${props => props.theme.space[3]}px;
+  right: ${props => props.theme.space[3]}px;
+  opacity: 1;
+`;
+
+const ResourceCard = styled.button<{ hasAccess?: boolean }>`
   position: relative;
-  align-items: center;
+  text-align: left;
   background: ${props => props.theme.colors.spotBackground[0]};
   transition: all 0.3s;
 
+  border: none;
   border-radius: 8px;
   padding: 12px 12px 12px 12px;
+  padding-block-start: 12px;
+  padding-block-end: 12px;
   color: ${props => props.theme.colors.text.main};
+  line-height: inherit;
+  font-size: inherit;
+  font-family: inherit;
```

Comment:
It looks the same without these properties :p

Reply:
I added these after changing from Div to Buttons, on Safari there was some weirdness going on with the Button inheriting a different font-family. Can test again though.

Reply:
Removing `font-family: inherit` still results in the Buttons using the system default font instead of `Ubuntu` :/

<img width="48%" alt="Screenshot 2024-08-14 at 12 38 36 PM" src="https://github.com/user-attachments/assets/8df257d8-9f5f-4a41-8ada-e6c6993d1ef6"/> <img width="48%" alt="Screenshot 2024-08-14 at 12 37 07 PM" src="https://github.com/user-attachments/assets/f841e5e5-53bf-4647-b703-018438129979"/>